ingest/rules/curate.smk

```diff
@@ -83,36 +87,37 @@ rule curate:
         sequence_field=config["curate"]["output_sequence_field"],
     shell:
         r"""
-        (cat {input.sequences_ndjson:q} \
-            | augur curate rename \
-                --field-map {params.field_map} \
-            | augur curate normalize-strings \
-            | augur curate transform-strain-name \
-                --strain-regex {params.strain_regex:q} \
-                --backup-fields {params.strain_backup_fields:q} \
-            | augur curate format-dates \
-                --date-fields {params.date_fields:q} \
-                --expected-date-formats {params.expected_date_formats:q} \
-            | augur curate parse-genbank-location \
-                --location-field {params.genbank_location_field:q} \
-            | augur curate titlecase \
-                --titlecase-fields {params.titlecase_fields:q} \
-                --articles {params.articles:q} \
-                --abbreviations {params.abbreviations:q} \
-            | augur curate abbreviate-authors \
-                --authors-field {params.authors_field:q} \
-                --default-value {params.authors_default_value:q} \
-                --abbr-authors-field {params.abbr_authors_field:q} \
-            | augur curate apply-geolocation-rules \
-                --geolocation-rules {input.all_geolocation_rules:q} \
-            | augur curate apply-record-annotations \
-                --annotations {input.annotations:q} \
-                --id-field {params.annotations_id:q} \
-                --output-metadata {output.metadata:q} \
-                --output-fasta {output.sequences:q} \
-                --output-id-field {params.id_field:q} \
-                --output-seq-field {params.sequence_field:q}
-        ) 2> {log:q}
+        exec &> >(tee {log:q})
+
+        cat {input.sequences_ndjson:q} \
+          | augur curate rename \
+              --field-map {params.field_map} \
```

Comment:
(non-blocking)

In the latest pathogen-repo-guide [style guide changes](https://github.com/nextstrain/pathogen-repo-guide/pull/86), I updated `{params.field_map}` with quoted interpolation in case you wanted to add that here. https://github.com/nextstrain/pathogen-repo-guide/commit/878898c0e19b0959061e17d8c9acdcb347601753